fontbe/src/avar.rs

```diff
@@ -25,8 +26,14 @@ fn to_segment_map(axis: &Axis) -> SegmentMaps {
     // because, per spec, "If the segment map for a given axis has any value maps,
     // then it must include at least three value maps: -1 to -1, 0 to 0, and 1 to 1"
     // so three value maps *must* produce identity.
+    //
+    // 0 entry avar may confuse some clients so emit a default one
```

Comment:
this change to fontbe::avar isn't really related to --emit-ir flag, is it?

Reply:
i see, it's to fix/workaround https://github.com/googlefonts/fontations/pull/367